quickassist/utilities/osal/src/linux/user_space/OsalThread.c

```diff
@@ -106,7 +106,7 @@ OSAL_PUBLIC OSAL_STATUS osalThreadCreate(OsalThread *thread,
             osalLog(OSAL_LOG_LVL_ERROR,
                     OSAL_LOG_DEV_STDOUT,
                     "\nosalThreadCreate:"
-                    "Failed to set the sched parameters atribute !!!\n");
+                    "Failed to set the sched parameters attribute!!!\n");
```

Comment:
Maybe remove the three exclamation marks :-)

Reply:
I've replaced all !!! and !! occurrences with ! and re-pushed.